src/app/lib/ledger.ts

```diff
@@ -1,11 +1,13 @@
 import { ContextSigner } from '@oasisprotocol/client/dist/signature'
```

Comment:
I think commits could be redone much better, e.g.:
- Add capacitor dependencies for mobile app support (package, yarn.lock)
- Differentiate USB and Bluetooth Ledger types (affects some existing tests)
- Add state for enumerating Bluetooth Ledger devices
- Add Bluetooth Ledger tests
- Add changelog fragment